Sources/StreamChatUI/ChatMessageList/Reactions/ChatReactionPickerBubbleView+Default.swift

```diff
@@ -5,7 +5,7 @@
 import StreamChat
 import UIKit
 
-open class ChatMessageDefaultReactionsBubbleView: ChatMessageReactionsBubbleView {
+open class ChatMessageDefaultReactionsBubbleView: ChatReactionPickerBubbleView {
```

Comment:
The name of this file is still weird 🤔 The name of the file should be the name of the class. And when using `+` the right side should contain the function or class name being extended. 

In this case the name of the file should be `ChatMessageDefaultReactionsBubbleView.swift`, currently is
`ChatReactionPickerBubbleView+Default.swift`. I don't see any `PickerBubbleView`, also the `+Default` should be there as well